tools/spotless/openremote-license-header.txt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright $YEAR, OpenRemote Inc.
+ *
+ * See the CONTRIBUTORS.txt file in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Adding a SPDX license identfier to the header also helps other software with identifying software licenses (ISO/IEC 5962:2021).

See: https://spdx.org/licenses/

```suggestion
  *
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
```